backend/app/core/mail.py

```
@@ -33,39 +35,83 @@ async def get_user_email(user_id: int, db: AsyncSession) -> Optional[str]:
+                
+            # Log successful send
+            success_msg = f"Email sent successfully to {to_email}"
+            if cc_emails:
+                success_msg += f", CC: {', '.join(cc_emails)}"
+            if bcc_emails:
+                success_msg += f", BCC: {len(bcc_emails)} recipient(s)"
+            if request_id:
+                success_msg += f" [Request ID: {request_id}]"
+            logger.info(success_msg)
+            
         except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)